vuepress/docs/next/docs/curate/bundle-version-types.md

```diff
@@ -0,0 +1,40 @@
+
+Some of the major improvements with the Docker-based bundles are:
+* Simplified bundle directory structure
+* Just one registry required with repositories created on the fly
+* Optimized file storage and transfer using a layered file-system, beneficial for large projects
+* Standardized bundle address format
+* Easier to modify, compose, and troubleshoot with decoupled processes 
+
+For specifications on Docker-based bundles, see the [Bundle Details](ecr-bundle-details.md) page. 
+### Bundle Comparison Table
+|Attribute** | [git-based bundles](../../../v7.0/docs/curate/ecr-bundle-details.md)| [docker-based bundles](ecr-bundle-details.md)|
+| :- | :-------------------- | :---------------------  |
+|`descriptorVersion`| v1| v5
+| Bundle specifications |Define with descriptor files | Build with [ent bundle CLI](../getting-started/ent-bundle.md) tool
+| Versioning |Bundle version set by git tags |Bundle version set by Docker Tags. For MFEs, the version defaults to 1.0.0 unless set in [entando.json](ecr-bundle-details.md). 
+|Plugin Ingress| Generated by Entando or set in the CRD | Generated by Entando or customize in the [bundle descriptor](ecr-bundle-details.md#entando-bundle-conventions), entando.json 
+|Identifiers| Designated Bundle ID and Plugin ID | Bundles, MFEs, & Plugins have unique global IDs|
+|Initialize from Entando Hub|NA |[Direct initialization from Hub](../../tutorials/create/mfe/react.md)(switch to simple widget as available)  with `ent init BUNDLE-NAME –from-hub`|
+|Bundle Creation | `ent prj` commands| New modular `ent bundle` commands
+|Deploy & Install| `ent ecr deploy` and `ent ecr install` | No change 
+|Export Bundle|`ent bundler`|`ent generate-cr` and export files 
+|JHipster Integrated Blueprint Project| `ent jhipster blueprint` creates [MFE & MS template project](../../tutorials/create/ms/generate-microservices-and-micro-frontends.md)| NA
+|APIs| NA| [Manage APIs with `ent bundle` commands](../getting-started/ent-api.md). MFEs can make API claims to MSs within the namespace
+||| MFEs support service discovery by using the property apiClaim
+|Services |Utilizes Dockerfiles or CRDs to specify services. Keycloak is integrated with the ent CLI | Default services now include PostgreSQL, MySQL, and Keycloak. Additional services can be added with the [`ent bundle svc` commands](../getting-started/ent-svc.md).
+|MFE custom UI| Add with manual scripts| Custom UI is auto-generated in FTL format
+|Data transfer |NA| New ways to pass values or information to widgets, bundles, or expose in the APIs by [utilizing context parameters, system parameters and the ext variable](ecr-bundle-details.md).
+|Bundle Thumbnail| Detailed inside the YAML bundle descriptor|Automatically setup with a jpg or png file named thumbnail in the bundle root folder.  e.g. thumbnail.png
+
+* Alternate page titles
+# Entando Bundle Version Comparison
```

Comment:
I prefer this one of the 3.  I don't love Version in the title for some reason (since bundles themselves have versions when published) but I don't have a better idea. In some ways this is a Feature Matrix like you get when you do insurance shopping. Entando Bundle Feature Matrix?

Reply:
How about Entando Bundle Transition to 7.1? 
Entando Bundle Transition from 7.0 to 7.1
How Entando Bundles changed from 7.0 to 7.1

Reply:
Feature Matrix for Bundle Transition?